compiler/parser/api.go

```diff
@@ -17,7 +19,7 @@ func ParseZed(filenames []string, src string) (interface{}, error) {
 	if err != nil {
 		return nil, ImproveError(err, src, srcInfo)
 	}
-	return p, nil
+	return ast.Seq(sliceOf[ast.Op](p)), nil
```

Comment:
Why not change the return type to ast.Seq instead?
```suggestion
	return sliceOf[ast.Op](p), nil
```

Reply:
Definitely.